solr/core/src/java/org/apache/solr/search/join/ChildFieldValueSourceParser.java

```diff
@@ -128,16 +97,7 @@ public String toString() {
     public SortField getSortField(boolean reverse) {
       final Type type = childField.getSortField(reverse).getType();
       return new ToParentBlockJoinSortField(
-          childField.getName(), type, reverse, parentFilter, childFilter) {
```

Comment:
I'm not sure where these lines go, but this is nice removal. 👍

Reply:
AFAICT these lines existed to compensate for the fact that SchemaField was not carried along to support marshalling SortFields. This PR introduces an alternative approach that makes this no longer necessary (and works equally well in distrib/standalone).